vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_authorization.h

```diff
@@ -25,30 +26,43 @@ class UpholdAuthorization {
 
   ~UpholdAuthorization();
 
-  void Authorize(
-      const base::flat_map<std::string, std::string>& args,
-      ledger::ExternalWalletAuthorizationCallback callback);
+  void Authorize(const base::flat_map<std::string, std::string>& args,
+                 ledger::ExternalWalletAuthorizationCallback callback) const;
+
+  void GetAnonFunds(
```

Comment:
This method (and `TransferAnonFunds`) are logically private, but I suppose that the unit tests want to call them directly? Should we just friend the unit test class instead of making them public? @emerick do you know if we have a preferred approach?

Reply:
That's right!
Apart from being called directly by the unit tests, there's really no good reason to make them `public`.
Actually we could keep them `private` and use [explicit template instantiation](https://godbolt.org/z/ezaMWe4vY) to be able to call them from the unit tests, but that might be overkill.

Reply:
Resolved in e7a597ba21dd2e98bb46cc6f8791c33c848e6525.